tools/validators/instance_validator/validate/telemetry_validator.py

```diff
@@ -19,23 +19,36 @@
 """
 
 import threading
+from typing import Dict
 
 from validate import field_translation as ft_lib
-from validate import telemetry
 from validate import message_filters
-from validate import telemetry_validation_reporting as tvr
+from validate import telemetry
+from validate import telemetry_validation_report as tvr
 
 DEVICE_ID = telemetry.DEVICE_ID
+DEVICE_NUM_ID = telemetry.DEVICE_NUM_ID
+GUID = 'guid'
 
 
 class TelemetryValidator(object):
   """Validates telemetry messages against a building config file.
 
-  Attributes;
-    entities: a dict with entity_name as a key and EntityInstance as value.
-    timeout: the max time the validator must read messages from pubsub.
-    is_udmi: true/false treat telemetry stream as UDMI
-    callback: the method called by the pubsub listener upon receiving a msg.
+  Attributes:
+    entities_with_translation: Mapping of entity codes to an EntityInstance
+      instances. that map 1:1 from a building config to a telemetry message.
+    timeout: The max time the validator must read messages from pubsub.
+    callback: The method called by the pubsub listener upon receiving a msg.
```

Comment:
They are initialized in __init__ and used in several methods. StartTimer() and AllEntitiesValidated() are two methods that use these variables.